docs/component-libraries.md

```diff
@@ -0,0 +1,15 @@
+# Component Libraries
+
+If you find yourself asking "which `<Button>` component should I use when working with the Calypso repo?", then this is the document for you!
+
+Admittedly, our components are quite disorganized and it’s not clear what should be used, and when. There are multiple versions of (allegedly) the same component, no design system, and no coherent design language to use in order to ensure we’re building a consistent user experience. Over the years, we’ve had different representations of a components team and initiatives to unify and use the same design language everywhere, but none of them really succeeded. 
+
+When figuring out which `<Button>` component to use, here are some guidelines you could consider:
+
+* `@wordpress/components` can be used for interfaces in the WordPress.org context (e.g. Block Editor).
+* `@automattic/components` or `client/components` can be used for interfaces in the WordPress.com context.
+* `client/blocks` represents more complex components that are usually composed from multiple UI components and are usually connected to the Redux state.
```

Comment:
Which might be the most appropriate link or entry point for each of these?

Reply:
I'd say https://wordpress.github.io/gutenberg for `@wordpress/components`.